usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/SimpleShellCommandTest.java

```diff
@@ -87,21 +90,24 @@
      * If not explicitly configured, the default assertion is a non-zero exit code.
      */
     @SetFromFlag("assertStatus")
-    ConfigKey<Map> ASSERT_STATUS = ConfigKeys.newConfigKey(Map.class, "assert.status",
-            "Assertions on command exit code", Maps.newLinkedHashMap());
+    ConfigKey<List<Map<String, Object>>> ASSERT_STATUS = ConfigKeys.newConfigKey(
```

Comment:
This change doesn't alter the syntax of the assertions in the Yaml, (except for the new SimpleShellCommand which it just brings in to line with the other tests), i.e.
```
assert:
- equals: true
```
But in fact any tests that were written before the new test assertions class were put in wouldn't work now (actually since https://github.com/apache/incubator-brooklyn/pull/1049) as they read
```
assert:
   equals: true
```
However I think there are very few such tests, as people had only started using the framework when 1049 was done.

At the same time it does seem a good improvement to make it work with a single map and wrap that in a list if need be.  (Is there code already somewhere that does that sort of thing?  How can you have a config key that might be a map or might be a list?)